compiler/src/dotty/tools/dotc/reporting/MessageRendering.scala

```diff
@@ -101,9 +100,12 @@ trait MessageRendering {
 
       math.min(currPad, minPad)
     }
+    val padding =
+      if (leastWhitespace - offset == pos.startColumn) pos.startColumnPadding
+      else " " * (leastWhitespace - offset)
```

Comment:
I don't understand why this is correct. Why not use startColumnPadding in currPad above?